### PR TITLE
make ap() and wr() work for py3

### PR DESCRIPTION
**Pull request by brettatoms**
This assumes that on Py3 if the `target` param on ap() and wr() is file-like then it is expecting a string rather than bytes.  This might not always be the case but it at least makes `ex('echo yolo') > sys.stdout` work.

### CODE REVIEW

File: shell/run_cmd.py [hunk 1]
```diff
@@ -113,13 +117,18 @@ def __gt__(self, target):
     def ap(self, target, source='stdout'):
         if source != 'stdout' and source != 'stderr':
             raise ValueError('unsupported source: {0}'.format(source))
+
+        out = getattr(self, source)()
+
         if isinstance(target, basestring):
             fd = open(target, 'ab')
             fd.write(getattr(self, source)())
```
Comment (houqp) [inline shell/run_cmd.py]: Could you replace getattr with out in this line too?

**Comment by houqp** [comment]
Hi @brettatoms , thanks for the patch :) Could you fix the travis build too?

**Comment by brettatoms** [comment]
I misunderstood what the real problem was.  I think this should fix it.

**Comment by houqp** [comment]
awesome, thanks for the fix :) will do a new release today.

**Pull request merged**